frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-install-single-namespace.spec.ts

```diff
@@ -93,19 +119,15 @@ xdescribe(`Interacting with a single namespace install mode Operator (${operator
   it(`displays details about ${operatorName} ${operatorInstance} instance on the "Details" tab`, () => {
     cy.log(`navigate to the "Details" tab`);
     cy.byTestOperandLink(operandLink).click();
-    cy.byTestSectionHeading('APIManagerBackup Overview').should('exist');
+    cy.byTestSectionHeading('Couchbase Cluster Overview').should('exist');
   });
 
-  it(`uninstalls the operator from ${testName} namespace`, () => {
-    cy.log('navigate to the operator uninstall modal in OperatorHub');
+  it(`uninstalls the Operator from ${testName} namespace`, () => {
```

Comment:
Same comment about running into the Cypress anti-pattern of [Having-tests-rely-on-the-state-of-previous-tests](https://docs.cypress.io/guides/references/best-practices.html#Having-tests-rely-on-the-state-of-previous-tests).